src/pre_process/include/1dHardcodedIC.fpp

```diff
@@ -1,5 +1,53 @@
 #:def Hardcoded1DVariables()
+
+!> @file 1DHardcodedIC.fpp
```

Comment:
confused on how exactly this works... it looks like it will do this for every hardcoded 1d case? we don't want that

Reply:
and obviously not every hardcoded case is supposed to work like this (with chemistry and so on). maybe i'm just misunderstanding how this file works.

Reply:
im not sure i understand what you mean